db/migrate/20201029194434_create_users.rb

```diff
@@ -0,0 +1,13 @@
+class CreateUsers < ActiveRecord::Migration[6.0]
+  def change
+    create_table :users do |t|
+      t.string :first_name
+      t.string :last_name
+      t.string :user_name
+      t.datetime :dob
```

Comment:
why do we need date of birth? is this for a happy birthday message we were thinking of doing in the app?

Reply:
Yeah that's right :)